WooCommerce/WooCommerceTests/ViewRelated/Orders/Payment Methods/PaymentMethodsViewModelTests.swift

```diff
@@ -135,18 +141,60 @@ final class PaymentMethodsViewModelTests: XCTestCase {
         await viewModel.markOrderAsPaidByCash(with: nil)
     }
 
+    func test_mark_order_as_paid_by_cash_then_order_status_and_payment_method_fields_updated() async {
+        // Given
+        let stores = MockStoresManager(sessionManager: .testingInstance)
+        let storage = MockStorageManager()
```

Comment:
Not necessarily for this PR, but since we have 20+ instances of `MockStorageManager()`, `siteID`, and others, we could make some `setUp` and `tearDown` in this test suite so each unit test is clearer 👍

Reply:
Good observation 👍 . Init and deinit of `MockStoresManager` and `MockStorageManage` can both be moved to setUp and tearDown. It's tougher with other parts of the tests since they differ slightly from one another.